### PR TITLE
Migrated rest UI handlers to Flask.

### PR DESCRIPTION
**Pull request by iamarshsingh**
1. commit_range
2. coverage_report
3. fuzzer_stats
4. gcd_redirector
5. performance_report
6. report_csv_failure
7. reproduce_tool handlers
8. revisions_info

Please have a look!

### CODE REVIEW

File: src/appengine/server_flask.py [hunk 1]
```diff
@@ -47,12 +57,22 @@
...
 from handlers.testcase_detail import update_issue
+from libs import helpers
 from metrics import logs
 from system import environment
 
 ndb_client = ndb.Client()
 
 
+class EmptyHandler(base_handler_flask.Handler):
+  """Empty hanlder for flask migration."""
```
Comment (oliverchang) [inline src/appengine/server_flask.py:68]: nit: spelling.

File: src/appengine/handlers/revisions_info.py [hunk 2]
```diff
@@ -51,7 +52,8 @@ def get(self):
       raise helpers.EarlyExitException('Failed to get component revisions.',
                                        400)
 
-    self.render('revisions-info.html',
-                {'info': {
-                    'componentRevisionsList': component_revisions_list
-                }})
+    return self.render(
+        'revisions-info.html',
+        {'info': {
```
Comment (oliverchang) [inline src/appengine/handlers/revisions_info.py:57]: nit:  this indentation looks a bit weird. Can you space things out a bit more suchj that this looks like:

```python
{
   'info': {
     'componentRevisionsList...':
   },
})
```

File: src/appengine/server_flask.py [hunk 3]
```diff
@@ -47,12 +57,22 @@
...
+
+  def get(self):
+    # TODO(singharshdeep): Remove routes based on this handler
+    # after flask migration.
+    raise helpers.EarlyExitException('URL not complete.', 404)
```
Comment (oliverchang) [inline src/appengine/server_flask.py:73]: Can you explain why this is needed?

Comment (iamarshsingh) [reply src/appengine/server_flask.py:73]: the middleware redirects on the basis of the first path before the second '/' so paths like '/corpus-backup/make-public' won't get redirected to flask but to webapp, so just to redirect these URLs to flask I am using this.
I also considered to modify the middleware, but as currently its working well and many handlers are already migrated, I didn't wanted to modify it. Also, this hack will removed in a few days.

File: src/appengine/handlers/commit_range.py [hunk 4]
```diff
@@ -15,11 +15,12 @@
 
 import json
 
+from flask import request
```
Comment (oliverchang) [inline src/appengine/handlers/commit_range.py:18]: Forgot to mention this, but there should be a newline in between third party dependencies and our own libraries.

Comment (iamarshsingh) [reply src/appengine/handlers/commit_range.py:18]: Sure, will fix this as part of next PR.

**Pull request merged**